lib/quadquizaminos/bottom.ex

```diff
@@ -24,13 +31,16 @@ defmodule Quadquizaminos.Bottom do
   end
 
   def complete?(bottom, row) do
+    ignore_result = vulnerable?(bottom, row)
```

Comment:
the variable `ignore_result` isn't that clear of what is doing. Since its supposed to return true or false if bottom has a vulnerable block. Maybe , you can name it `has_vulnerable_block?`

Reply:
I'll name it as `has_vulnerable?` ,  so that line will now appear as `has_vulnerable? = has_vulnerablility_block?(bottom, row)`